ui-tests/cypress/support/pageObjects/pages/management/AddPage.js

```diff
@@ -0,0 +1,147 @@
+import {DATA_TESTID, htmlElements} from "../../WebElement.js";
+
+import Content from "../../app/Content.js";
+import AppPage from "../../app/AppPage.js";
+import ManagementPage from "./ManagementPage.js";
+
+export default class AddPage extends Content {
+    pageForm = `${htmlElements.div}[${DATA_TESTID}=add_PagesAddPage_Grid]`;
+
+    inputTitle = `${htmlElements.input}[name="titles.en"]`;
+    inputTitleIt = `${htmlElements.input}[name="titles.it"]`;
+    inputCode = `${htmlElements.input}[name=code]`;
+    pageTreeSelector = `${htmlElements.div}[${DATA_TESTID}=PageForm__PageTreeSelector]`;
+    pageTemplateSelector = `${htmlElements.select}[name=pageModel]`;
+
+    // SEO
+    seoInfoContainer = `${htmlElements.div}[id=basic-tabs]`;
+    seoInfoTabs = `${htmlElements.ul}[${DATA_TESTID}=common_SeoInfo_Tabs]`;
+    seoDescription = `${htmlElements.input}[name="seoData.seoDataByLang.en.description"]`
+    seoDescriptionIt = `${htmlElements.input}[name="seoData.seoDataByLang.it.description"]`
+    seoKeywords = `${htmlElements.input}[name="seoData.seoDataByLang.en.keywords"]`
+    seoKeywordsIt = `${htmlElements.input}[name="seoData.seoDataByLang.it.keywords"]`
+    seoFriendlyCode = `${htmlElements.input}[name="seoData.seoDataByLang.en.friendlyCode"]`
+    seoFriendlyCodeIt = `${htmlElements.input}[name="seoData.seoDataByLang.it.friendlyCode"]`
+
+    // Meta
+    metatagKey = `${htmlElements.input}[name="metakey"]`
+    metatagType = `${htmlElements.input}[name="metatype"]`
+    metatagValue = `${htmlElements.input}[name="titles.en"]`
+
+    // buttons
+    saveButton = `${htmlElements.button}[${DATA_TESTID}="save-page"]`;
+    saveAndDesignButton = `${htmlElements.button}[${DATA_TESTID}="common_PageForm_Button"]`;
+
+    getPageForm () {
+        return this.parent.get()
+                .find(this.pageForm);
+    }
+
+    getSeoContainer () {
+        return this.parent.get()
+               .find(this.seoInfoContainer);
+    }
+
+    getSeoTabs() {
+        return this.getSeoContainer()
+               .children(this.seoInfoTabs);
+    }
+
+    selectSeoLanguage(langOrder) {
+        this.getSeoTabs().find(htmlElements.a).eq(langOrder).click();
+        cy.wait(1000);
+    }
+
+    getInputTitle(lang) {
```

Comment:
Maybe you can create a multi language object or just put the selector here adding the parameter as part of it, instead of using an if to select one of the two values; we might have additional languages in the future and this would not be the best approach

Reply:
I made it simpler by adding a single selector for each field and replace `{code}` by the code sent by the spec file

Reply:
that's where I would have used the private modifier 😝